consensus/consensus-types/src/block.rs

```diff
@@ -103,6 +103,23 @@ impl<T> Block<T> {
     pub fn timestamp_usecs(&self) -> u64 {
         self.block_data.timestamp_usecs()
     }
+
```

Comment:
why the move :O

Reply:
original it stays in an impl block with unnecessary generic constraint